lib/carrierwave/storage/aws_file.rb

```diff
@@ -60,6 +60,10 @@ def copy_to(new_path)
         bucket.object(new_path).copy_from(copy_source: "#{bucket.name}/#{file.key}")
       end
 
+      def signed_url(options = {})
+        uploader.sign_urls[:signer].call(public_url, options)
```

Comment:
Will the `sign_urls` option always be a hash? If so we should use `sign_urls.fetch(:signer)` to prevent `undefined method` errors when the key is missing. That said, it seems like the option could just be a lambda/proc instead.